docs/extensibility/resources.md

```diff
@@ -106,6 +106,10 @@ The `details` section defines the display structure for the details page. It con
 - **widget** - optional widget to render the defined entry. By default the value is displayed verbatim. For more information about the available widgets, see [Display widgets](display-widgets.md).
 - **valuePreprocessor** - name of [value preprocessor](#value-preprocessors),
 - **formula** - optional formula used to modify data referred to by the `path` property. To learn more about using formulas, see [JSONata](https://docs.jsonata.org/overview.html).
+- **visibility** - by default all fields are visible; however **visibility** property can be used to control a single item display.
```

Comment:
JSONata tested successfully for Panel children, doesn't work for other components:
CodeViewer - for `"path": "spec.files",` requires `"visibility": "$exists(data.spec.files)"`
Text - for `"path": "kind"` requires `"visibility": "$exists(data.kind)"`
same for the widgets in the header
etc.